__tests__/render.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import App from '../components/App.jsx';
+
+
+describe('First React component test with Enzyme', () => {
+   it('renders without crashing', () => {
+
+    shallow(<App />)    );
+});
+
+describe('App mount', () => {
+  it('App mounts', () => {
+
+    mount(<App />);    });
```

Comment:
In the 15th line has an extra parenthesis, semicolon and brackets after App component.
This test has no expectation `expect <something> to be <something>`